src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -61,43 +56,13 @@ function SelectedRevisionItem({
   const itemDate = new Date(item.push_timestamp * 1000);
   const { removeCheckedRevision } = useCheckRevision(searchType);
   const { deleteSelectedRevisions } = useSelectedRevisions();
-  const selectedRevisions = useAppSelector(
-    (state) => state.selectedRevisions.revisions,
-  );
-  const selectedFramework = useAppSelector((state) => state.framework);
-  const navigate = useNavigate();
-  const prevRevRef = React.useRef<RevisionsList[]>([]);
-  const location = useLocation();
-  const view = location.pathname == '/' ? searchView : compareView;
-  //hide the close icon for the selected revisions in compare view
+
+  //hide the close icon for the selected revisions in edit view
   const iconClassName =
-    !formIsDisplayed && view !== searchView
+    !formIsDisplayed && isEditable
       ? 'icon icon-close-hidden'
       : 'icon icon-close-show';
 
-  const updateRevsInHash = () => {
-    const revs = selectedRevisions.map((rev) => rev.revision);
-    const repos = selectedRevisions.map((rev) => repoMap[rev.repository_id]);
-
-    navigate({
-      pathname: '/compare-results',
-      search: `?revs=${revs.join(',')}&repos=${repos.join(',')}&framework=${
-        selectedFramework.id
-      }`,
-    });
-  };
-
-  React.useEffect(() => {
-    if (
-      prevRevRef.current !== selectedRevisions &&
-      selectedRevisions.length > 0
-    ) {
-      updateRevsInHash();
-    }
-
-    prevRevRef.current = selectedRevisions;
-  }, [selectedRevisions]);
-
```

Comment:
I think that removing this breaks some functionality (updating the parameter in the URL when changing the revisions), but I'm fine with removing it here because I believe it wasn't the right place to do it anyway. We'll add it back in a future PR.

Reply:
Yes, I'll create a new PR to address this.